src/mgr/MgrStandby.cc

```diff
@@ -160,8 +161,32 @@ void MgrStandby::send_beacon()
   auto addr = available ? active_mgr->get_server_addr() : entity_addr_t();
   dout(10) << "sending beacon as gid " << monc.get_global_id()
 	   << " modules " << modules << dendl;
-
+  
+  const char *host_ip = NULL;
   map<string,string> metadata;
+  auto addr_standby = monc.get_my_addr();
+
+  switch(addr_standby.get_family()) {
+  case AF_INET:
+    char addr_buf_inet[INET_ADDRSTRLEN];
```

Comment:
Defining the buffer inside the switch `{}` and then using the host_ip pointer to it from outside the `{}` seems incorrect -- I think the compiler is allowed to re-use that space after the end of the `{}`